main.go

```diff
@@ -2,19 +2,57 @@ package main
 
 import (
 	"fmt"
+	"github.com/fatih/color"
 	"io/ioutil"
 	"os"
 	"regexp"
 	"strings"
 )
 
+const fileName = "emails"
+
 func validateEmail(email string) bool {
 	emailRegExp, _ := regexp.Compile("^[a-zA-Z0-9.!#$%&'*+\\/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$")
 
 	return emailRegExp.MatchString(email)
 }
 
-const fileName = "emails"
+func doEmailExists(emailList []string, n string) bool {
+	for _, e := range emailList {
+		if n == e {
+			return true
+		}
+	}
+	return false
+}
+
+func printList(emailList []string) {
+	fmt.Println("E-mail list")
+	for i, email := range emailList {
+		fmt.Println(i+1, ")", email)
+	}
+}
+
+func printHighlightedLIst(emailList []string, selectedEmail string, textColor string) {
+	colors := map[string]color.Attribute{
+		"red":   color.FgHiRed,
```

Comment:
Nice but this could be remove because we're not using it!

Reply:
Yeah I saw that, but maybe it can be kept until "remove email" action is implemented and the red color can be used?

Reply:
Sure, that's a valid point! 👍